docs/explanations/audit-scanner/policy-reports.md

```diff
@@ -43,140 +43,174 @@ The audit results generated by the scanner include:
+  uid: 009805e4-6e16-4b70-80c9-cb33b6734c82
 summary:
   error: 0
-  fail: 0
-  pass: 6
+  fail: 1
+  pass: 0
   skip: 0
   warn: 0
 ```
 
-## Namespace-Specific Audit Results example
+## ClusterPolicyReport example
 
-In this example, the audit scanner evaluated many policies on resources within the `default` namespace.
-The results indicate that certain resources failed the validation for the `cap-no-privilege-escalation` policy.
-Others passed the validation for the `cap-do-not-run-as-root` policy.
-The `summary` section shows the number of failures and passes.
+The following example shows a `ClusterPolicyReport` for the `Namespace` resource `default`.
+The report indicates that the resource has failed the `safe-annotations` ClusterAdmissionPolicy validation.
 
 ```yaml
 apiVersion: wgpolicyk8s.io/v1beta1
-kind: PolicyReport
+kind: ClusterPolicyReport
 metadata:
-  creationTimestamp: "2023-07-10T19:28:05Z"
-  generation: 4
+  creationTimestamp: "2024-02-28T14:44:37Z"
+  generation: 3
   labels:
     app.kubernetes.io/managed-by: kubewarden
-  ...
+  name: 261c9492-deec-4a09-8aa9-cd464bb4b8d1
+  ownerReferences:
+    - apiVersion: v1
+      kind: Namespace
+      name: default
+      uid: 261c9492-deec-4a09-8aa9-cd464bb4b8d1
+  resourceVersion: "2403034"
+  uid: 20a3d00e-e955-4f21-a887-317d40f3f052
 results:
-- message: one of the containers has privilege escalation enabled
-  policy: cap-no-privilege-escalation
-  ...
-  resourceSelector: {}
-  resources:
-  - apiVersion: apps/v1
-    kind: Deployment
-    name: nginx-deployment
-    namespace: default
-    ...
-  result: fail
-  rule: no-privilege-escalation
-  source: kubewarden
-  timestamp:
-    nanos: 0
-    seconds: 1689017383
-- policy: cap-do-not-run-as-root
-  ...
-  resourceSelector: {}
-  resources:
-  - apiVersion: apps/v1
-    kind: Deployment
-    name: nginx-deployment
-    namespace: default
-    ...
-  result: pass
-  rule: do-not-run-as-root
-  source: kubewarden
-  timestamp:
-    nanos: 0
-    seconds: 1689017383
-...
+  - category: Resource validation
+    message: "The following mandatory annotations are not allowed: owner"
+    policy: clusterwide-safe-annotations
+    properties:
+      policy-resource-version: "2396437"
+      policy-uid: 46780d6e-e51a-4d65-8572-a6af01380aa7
+      validating: "true"
+    resourceSelector: {}
+    result: fail
+    scored: true
+    severity: low
+    source: kubewarden
+    timestamp:
+      nanos: 0
+      seconds: 1709294251
+scope:
+  apiVersion: v1
+  kind: Namespace
+  name: default
+  resourceVersion: "37"
+  uid: 261c9492-deec-4a09-8aa9-cd464bb4b8d1
 summary:
   error: 0
-  fail: 8
-  pass: 10
+  fail: 1
+  pass: 0
   skip: 0
   warn: 0
 ```
+
+## Policy Reporter UI
+
+The Policy Reporter is shipped as a subchart of `kubewarden-controller`.
+Refer to the [Audit Scanner Installation](../../howtos/audit-scanner)
+page for more information.
+
+The Policy Reporter UI provides a dashboard showcasing all violations
```

Comment:
```suggestion
The Policy Reporter UI provides a dashboard showing all violations
```

Questions about idiomatic English again. What's a 'showcase'. Would the audience always understand that? https://dictionary.cambridge.org/dictionary/english/showcase